### DISCUSSION TITLE
ESPHome config question

### DISCUSSION THREAD

**Discussion by stigory** [pio2398/W4G2#3] [General]
- lambda: |-
          UARTDebug::log_hex(direction, bytes, ' ');

Is this correct syntax?

**Comment by pio2398** [comment]
I think so. I am using this config for uart:
```
uart:
  id: mod_bus
  tx_pin: PA23
  rx_pin: PA18
  baud_rate: 9600
  debug:
    direction: BOTH
    dummy_receiver: true
    after:
      delimiter: [0xF4, 0xFB]
    sequence:
      - lambda: |-
          UARTDebug::log_hex(direction, bytes, ' '); //log the data
```

Although, all after debug is not required. Just that should give more useful logs for debugging.
